### DISCUSSION TITLE
Lineage Tracking for PySpark jobs on Databricks

### DISCUSSION THREAD

**Discussion by 97nitt** [AbsaOSS/spline-spark-agent#362] [Q&A] [answered]
[This doc](https://github.com/AbsaOSS/spline-getting-started/tree/main/spline-on-databricks) indicates that a workaround is required to enable lineage tracking when running _Notebooks_ on Databricks (adding a couple lines of Scala to the top of each notebook). 

Is something similar required when running PySpark jobs (via "Python" or "Spark Submit" jobs) on Databricks? I'll be attempting this in the next couple weeks, just wondering how easy/hard/impossible this is going to be...

**Comment by wajda** [comment, marked as answer]
The mentioned workaround is related the Databricks notebook. When using `pyspark`, `spark-shell` or `spark-submit` there shouldn't be any issues.

**Reply by 97nitt** [reply]
Excellent, thanks for the quick response.
